wherehows-web/app/components/dataset-compliance.js

```diff
@@ -7,14 +7,21 @@ const {
   get,
   isBlank,
   setProperties,
-  getWithDefault
+  getWithDefault,
+  String: { htmlSafe }
 } = Ember;
 
 // TODO: DSS-6671 Extract to constants module
 const missingTypes = 'Looks like some fields may contain privacy data ' +
   'but do not have a specified `Field Format`?';
 const successUpdating = 'Your changes have been successfully saved!';
 const failedUpdating = 'Oops! We are having trouble updating this dataset at the moment.';
+const hiddenTrackingFieldsMsg = htmlSafe(
```

Comment:
Interesting, I'm curious why they said it's not recommended, especially for the case when the string isn't user generated, or contains no user generated content (as is this case)? 
A partial is also a good approach, although I initially felt it was overkill to create a new file to render 4 lines.
I'll look this up, thanks for the feedback @guangyi :)